Justfile

```diff
@@ -32,8 +32,31 @@ format-generated-code:
   #!/usr/bin/env -S nix shell --accept-flake-config 'github:freckle/flakes?dir=main#fourmolu-default' --command bash
   fourmolu -i recurly-client/src --unsafe
 
-# Use version.txt as the version in the .cabal file
-set-package-version:
-  #!/usr/bin/env bash
+patch-cabal-file:
+  #!/usr/bin/env -S nix shell 'nixpkgs#haskellPackages.cabal-fmt' --command bash
```

Comment:
It seems like this means you are meant to run `just` not in a nix shell, then this bit will be run in a nix shell. Why wouldn't nix users just (heh) be using a nix shell all the time, even when invoking `just` such that a normal recipe without this custom shebang would behave correctly?

Reply:
I think that was lazy before I had defined a flake; will move it all into the flake.